### ISSUE TITLE
Add momentum to the LearningRateMonitor

### ISSUE THREAD

**Issue by jeremyjordan** [PyTorchLightning/pytorch-lightning#4354]
## 🚀 Feature

We should add a flag to optionally log the momentum value (for optimizers that have the attribute) in our LearningRateMonitor.

### Motivation

Some learning rate schedulers (eg. [OneCycle](https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.OneCycleLR)) also adjust the momentum value when updating the learning rate. 

### Pitch

For each scheduler's optimizer, we can check to see if it has a momentum attribute. If so, we log the value in addition to the learning rate.

**Comment by jeremyjordan** [comment]
cc @rohitgr7 and @Borda - what do you think? I'm happy to open a PR for this.

**Comment by rohitgr7** [comment]
I like it 👍
